app/controllers/devices_controller.rb

```diff
@@ -0,0 +1,54 @@
+class DevicesController < ApplicationController
+
+  def index
+    @devices = Device.all
+  end
+
+  def new
+    @device = Device.new
+    @house = House.find(params[:house_id])
```

Comment:
First of all, most probably you want to give access to houses of current_user (the one that is logged in right now)
Second, you can DRY a little the code if you will write a protected method like this:
```
def current_house
  current_user.houses.find(params[:house_id])
end
```
in this way the code is more readable